### PR TITLE
mrc-2680: Return dummy warnings from endpoints

### PR DESCRIPTION
**Pull request by r-ash**
This PR will
* Always return static warnings from `/model/result`, `/calibrate/result` and `/validate/options` endpoints

This includes schema changes from #304

### CODE REVIEW

File: inst/schema/ModelResultResponse.schema.json [hunk 1]
```diff
@@ -6,7 +6,13 @@
     "complete": {
       "type": "boolean",
       "enum": [ true ]
+    },
+    "warnings": {
```
Comment (hillalex) [inline inst/schema/ModelResultResponse.schema.json:10]: I suppose we could actually factor out this shared block into a single schema `Warnings.schema.json`

File: R/run_model.R [hunk 2]
```diff
@@ -95,7 +95,21 @@ process_result <- function(model_output) {
...
+         list(
+           text = scalar("ART coverage greater than 100% for 10 age groups"),
+           locations = list(scalar("model_calibrate"))
+         ),
+         list(
+           text = scalar("Prevalence greater than 40%"),
+           locations = list(scalar("model_calibrate"), scalar("review_output"))
+         ),
+         list(
+           text = scalar("ART coverage greater tha 100%"),
```
Comment (Nicolas-Dolan) [inline R/run_model.R:109]:
```suggestion
           text = scalar("ART coverage greater than 100%"),
```

**Review by Nicolas-Dolan — APPROVED** [review]

**Comment by EmmaLRussell** [comment]
Looks good but build is failing

**Comment by hillalex** [comment]
Sorry, meant to do this in a patch branch, but pulled out Warning into it's own schema: https://github.com/mrc-ide/hintr/pull/305/commits/96c80a4d738596b9e1a1c4d1497042c4d4fadff4

**Comment by r-ash** [comment]
#308 includes these changes

**Pull request closed without merging**